.github/workflows/publish-containers.yml

```diff
@@ -0,0 +1,131 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
I don't think we're using the `v` prefix any more? cc @maarcingebala

Reply:
We're not. Can we change it here or should we start using the `v` prefix for future releases instead? I don't have much preference, but the format without the prefix seems a bit cleaner to me.

Reply:
I will change it no problem 👍